easybuild/easyconfigs/l/libjxl/libjxl-0.3.7-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'libjxl'
+version = '0.3.7'
+local_lodepng_version = "48e5364"
+local_skcms_version = "6437475"
+
+homepage = 'https://github.com/libjxl/libjxl'
+
+description = """JPEG XL image format reference implementation"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+toolchainopts = {'optarch': False}  # optarch causes some test failures
+
+github_account = 'libjxl'
+source_urls = [GITHUB_SOURCE]
+sources = [
+    'v%(version)s.tar.gz',
+    {'filename': 'lodepng-%s.tar.gz' % local_lodepng_version,
+     'download_filename': local_lodepng_version,
+     'source_urls': ['https://github.com/lvandeve/lodepng/tarball/'],
+     'extract_cmd': 'tar -C libjxl-%(version)s/third_party/lodepng --strip-components=1 -xf %s'},
+    {'filename': 'skcms-%s.tar.gz' % local_skcms_version,
+     'download_filename': '%s.tar.gz' % local_skcms_version,
+     'source_urls': ['https://skia.googlesource.com/skcms/+archive/'],
+     'extract_cmd': 'tar -C libjxl-%(version)s/third_party/skcms --strip-components=0 -xf %s'},
+
+]
+patches = ['libjxl-%(version)s_openexr-3.0_compat.patch']
+checksums = [
+    '8318a5d1ce82e5683a52512b6f242d2b33a1191e89fc76f7c0ead0a1d60f6c60',  # v0.3.7.tar.gz
+    'f38176fc103fe1f6d23ba6addd5b14e6a54d546dfaa64663306acfe7b6d912ea',  # lodepng-48e5364.tar.gz
+    '1be9a0e16cdd1d2749c8111b862c496100ec959d2efaea7996eee86ef796fddd',  # skcms-6437475.tar.gz
```

Comment:
@Micket checksum problem?

Reply:
Not sure why the first tarball i managed to get a hold off had a different checksum. Fixed